### DISCUSSION TITLE
Disable asset file output hash

### DISCUSSION THREAD

**Discussion by kevinwarne** [vikejs/vike#1579] [Discussion]
Hello! First, vike is amazing and I appreciate all the work being done on the project. I did have a question about potentially disabling the hashing specifically on entry files.

The reason I need this is that I have a very large (mostly static) application requiring thousands of pages to get pre-rendered at build time. So in order to accommodate this, I batch our build into dozens of parallel jobs, each meant to output a different set of prerendered pages. In order to avoid hashing conflicts between jobs I am relying on entry point invalidations once all the jobs finish building and publishing.

It's possible there is a cleaner way to accomplish parallel prerendering (e.g. without regenerating the app chunks every time) in which case I could leave the file hashing in place.

Currently if I disable asset file hashing with ``` assetFileNames: `${assetDir}/assets/[name].[ext]` ```. I get the following error:

Setting config.build.rollupOptions.output.assetFileNames is currently forbidden. (It's possible to support, thus contact a maintainer if you need this.)

Any guidance here would be great!

**Comment by brillout** [comment]
I think it's possible to support, but it's far from trivial. (See comments above the assertion.)

I wonder if any Vite-powered framework or Next.js supports this?

To be honest an issue is that, so far, you're the only one needing this while [there are a lot of high priorities](https://github.com/vikejs/vike/labels).

> Hello! First, vike is amazing and I appreciate all the work being done on the project.

Thanks! I appreciate the kind words.
